src/hooks/events.ts

```diff
@@ -16,10 +16,7 @@ export interface VenueEventsData {
   events: WorldEvent[];
 }
 
-export const useSpaceEvents = ({
```

Comment:
Refactored into two methods: `useSpaceEvents` and `useWorldEvents`. Makes the APIs simpler and cleaner.